browser/brave_ads/tabs/ads_tab_helper.cc

```diff
@@ -32,6 +34,10 @@ constexpr char16_t kGetDocumentHTMLScript[] =
 
 constexpr char16_t kGetInnerTextScript[] = u"document?.body?.innerText";
 
+bool UserHasOptedInToBraveRewards(const PrefService* prefs) {
```

Comment:
this method doesn't seem useful to me, why would we wrap the pref check like this?